MapboxNavigation/NavigationView.swift

```diff
@@ -123,7 +129,19 @@ open class NavigationView: UIView {
         view.cancelButton.addTarget(self, action: Actions.cancelButton, for: .touchUpInside)
         return view
     }()
-    
+
+    var endOfRouteView: UIView? {
+        didSet {
+            if let active: [NSLayoutConstraint] = constraints(affecting: oldValue) {
+                NSLayoutConstraint.deactivate(active)
+            }
+
+            oldValue?.removeFromSuperview()
+            if let eor = endOfRouteView { addSubview(eor) }
```

Comment:
```suggestion
            if let localEndOfRouteView = endOfRouteView {
                addSubview(localEndOfRouteView) 
            }
```

Reply:
Good call — I ended up addressing it slightly differently. Please take another look.